package.json

```diff
@@ -16,6 +16,8 @@
     "webpack": "1 || ^2.1.0-beta"
   },
   "devDependencies": {
+    "babel-core": "^6.0.0",
```

Comment:
If we are going to add it as a `devDependency` which is fine as we only support `^6.0.0` it needs to be removed as a `peerDependency`